src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -64,11 +66,13 @@
  * <li>
  * Property {@code excludeClassesRegexps} - Specify user-configured regular
  * expressions to ignore classes.
+ * Type is {@code Regexp}.
  * Default value is {@code ^$}.
  * </li>
  * <li>
  * Property {@code excludedPackages} - Specify user-configured packages to ignore.
  * All excluded packages should end with a period, so it also appends a dot to a package name.
+ * Type is {@code StringCollection}.
```

Comment:
I think it is better to keep custom types with lowercase first letter - `stringCollection`.